pdk/1.4/modules/ROOT/pages/policies-pdk-configure-features-violations.adoc

```diff
@@ -0,0 +1,49 @@
+= Reporting Policy Violations
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+NOTE: For an example of reporting policy violations, see https://github.com/mulesoft/pdk-custom-policy-examples/blob/support/{template-policies-url-ver-var}/stream-payload/README.md[Stream Payload Policy].
+
+PDK provides functions to report a policy violation. Reported policy violations appear in Anypoint Monitoring.
+
+A request can only have one active policy violation. If a previous policy reports a policy violation, generating a new policy violation overrides the previous violation. If multiple policies generate policy violations for a single request, only one policy violation appears in Anypoint Monitoring. 
```

Comment:
```suggestion
Only one policy violation can be active for a request at a time. When a new violation occurs, it replaces any existing violation. If multiple policies generate violations, only the most recent one appears in Anypoint Monitoring.
```
This sentence is a bit repetitive, suggesting a rewording. Please check if it remains accurate.